### ISSUE TITLE
Figure out configuration XML in Java

### ISSUE THREAD

**Issue by djw8605** [scitokens/docker-scitokens-java#3]
When we left off, Jeff was debugging why the issuer in the container is not finding the configuration XML.  After that we will want to make sure that you can register the issuer with cilogon / comanage to authenticate.

**Comment by marshallmcdonnell** [comment]
I don't know if it is related but we are hitting the same issue trying to use this repository to spin up a SciTokens server with docker-compose.

What we have seen is the following error message:
```
scitokens-tomcat_1  | edu.uiuc.ncsa.security.core.exceptions.MyConfigurationException: Error: No configuration found. Cannot configure the server.
scitokens-tomcat_1  |     at edu.uiuc.ncsa.myproxy.oa4mp.server.servlet.AbstractBootstrapper.getConfigurationLoader(AbstractBootstrapper.java:36)
scitokens-tomcat_1  |     at edu.uiuc.ncsa.security.servlet.Bootstrapper.contextInitialized(Bootstrapper.java:39)
scitokens-tomcat_1  |     at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4699)
scitokens-tomcat_1  |     at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5165)
scitokens-tomcat_1  |     at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
scitokens-tomcat_1  |     at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:743)
scitokens-tomcat_1  |     at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:719)
scitokens-tomcat_1  |     at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:714)
scitokens-tomcat_1  |     at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:970)
scitokens-tomcat_1  |     at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1841)
scitokens-tomcat_1  |     at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
scitokens-tomcat_1  |     at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
scitokens-tomcat_1  |     at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
scitokens-tomcat_1  |     at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
scitokens-tomcat_1  |     at java.base/java.lang.Thread.run(Thread.java:829)
```

Believe this is coming from this line here in the OA4MP project:
https://github.com/ncsa/OA4MP/blob/5.2-sci-auth/oa4mp-server-api/src/main/java/edu/uiuc/ncsa/myproxy/oa4mp/server/servlet/AbstractBootstrapper.java#L33-L39

Which points to this name that needs to exist in the config file, which appears to map to the  `"oa4mp:server.config.name"` in the configuration file:
https://github.com/ncsa/OA4MP/blob/5.2-sci-auth/oa4mp-server-api/src/main/java/edu/uiuc/ncsa/myproxy/oa4mp/server/servlet/AbstractBootstrapper.java#L18-L22

Finally, it looks like that is defined here:
https://github.com/scitokens/docker-scitokens-java/blob/master/scitokens-server/web.xml#L234-L237

Can't tell what the actual issue is yet either but just wanted to elaborate to see if we can help out.

Thanks!

**Comment by jjg-123** [comment]
Resolved. Missing library file and a wrong name for a parameter.  At this point it comes up, but we need to test it further.

**Comment by marshallmcdonnell** [comment]
Awesome news, @jjg-123!

One quick question though, is this being resolved in another repo or in the current one?
I don't see an open PR to close this issue in the current repo.
If it is in another repo, can you provide the link to that PR?

Happy to test the fix in the coming week!

Thanks!

**Comment by marshallmcdonnell** [comment]
@djw8605 Sorry, just to check, where is the work that fixed this issue?
The only changes I have seen in this repo are the following:
https://github.com/scitokens/docker-scitokens-java/commit/83cbaa24b858375ca3ce0842140846d1b71c9b49

Was that the work that fixed the problem?
Thanks

**Comment by djw8605** [comment]
Hi, sorry meant to comment when I closed the issue.  The repo is not in a "usable" state as we heavily rework how the issuer works.  The original issue was indeed fixed in 83cbaa2.

**Comment by marshallmcdonnell** [comment]
Awesome, thanks!
